nixos/modules/services/networking/networkmanager.nix

```diff
@@ -603,61 +697,68 @@ in
       wantedBy = [ "multi-user.target" ];
       before = [ "network-online.target" ];
       after = [ "NetworkManager.service" ];
+
       script = let
-        path = id: "/run/NetworkManager/system-connections/${id}.nmconnection";
+        substProfile = profile: let
+          ini = pkgs.formats.ini { };
+
+          connectionPath = id:
+            "/run/NetworkManager/system-connections/${id}.nmconnection";
+
+          inputPath = ini.generate (escapeShellArg profile.n) profile.v;
```

Comment:
Error here, should be `lib.ini`.

Reply:
caught it, just haven't been back at my computer for a bit